bioptim/examples/torque_driven_ocp/example_soft_contact.py

```diff
@@ -49,7 +50,9 @@ def prepare_single_shooting(
     biorbd_model = biorbd.Model(biorbd_model_path)
 
     # Dynamics
-    dynamics = Dynamics(DynamicsFcn.TORQUE_DRIVEN, implicit_dynamics=False, implicit_soft_contacts=False)
+    dynamics = Dynamics(
+        DynamicsFcn.TORQUE_DRIVEN, multibody_dynamics=MultiBodyDynamics.EXPLICIT, implicit_soft_contacts=False
```

Comment:
Please make this:
`DynamicsFcn.TORQUE_DRIVEN, multibody_dynamics=Dynamics.EXPLICIT, soft_contacts_dynamics=Dynamics.EXPLICIT`
with default for `soft_contacts_dynamics` being the same as `multibody_dynamics`

The easiest way to do that is to set the default value to `None`, if None then equal else use the not Noned value